fake-koji/src/main/java/org/fakekoji/xmlrpc/client/JavaClient.java

```diff
@@ -52,36 +58,44 @@ public static void main(String[] args) throws MalformedURLException, XmlRpcExcep
 //        Object result = client.execute("sample.sum", params);
 //        int sum = ((Integer) result);
 //        System.out.println("The sum is: " + sum);
-        Integer packageId = (Integer) execute(Constants.getPackageID, "java-1.8.0-openjdk");
+        final XmlRpcRequestParamsBuilder paramsBuilderFirst = new XmlRpcRequestParamsBuilder();
```

Comment:
Just a tip!
I like to write builders so you can chain them like 
```SomeObject o = new SomeObjectBuilder().withName("blabla").build()``` 
You can easily achieve that by returning this from `with` method, like
```
class XmlRpcRequestParamsBuilder {
...
public XmlRpcRequestParamsBuilder setPackageName(String name) {
    this.name = name;
    return this;
}
...
}
```
Note that `with` prefix is just convention I like, I'm ok with `set` if it's convention for this project.